### ISSUE TITLE
I don't get correct VectorDrawable files with the tools 'L' is inserted at a wrong place.

### ISSUE THREAD

**Issue by henningBunk** [inloop/svg2android#62]
Hi,
I tried to convert SVG files which I got out of PDF files. I used Poppler for converting. The path data looks like that:

`d="M 6.808594 26.90625 C 3.246094 25.6875 1.097656 23.179688 1.097656 20.203125 ...`

I get the following path data with this tool:

`android:pathData="M 6.808594 L 26.90625 C 3.246094 25.6875 1.097656 23.179688 1.097656 20.203125 ...`

The `L` looks very odd at that place. When I remove the line 347 of main.js (`t = (bigM ? DRAW_LINE.toUpperCase() : DRAW_LINE) + " " + t;`) the L isn't added and it works for me.
Aren't my SVG files following the correct standards or does this tool require a certain formatting?

Bye!

**Comment by yuraj11** [comment]
I have commented that line for now. I think It was a bit wrong there but I am not sure which SVGs this change will broke. (I should make some batch svg tests )

**Comment by fredpyo** [comment]
I found a several paths that break with this change. Don't know how easy to replicate, but entering text and converting it to shapes produces for example this:

https://gist.github.com/fredpyo/69be34607143cc6c7d5a8e9a16bad6b3

The path begins with:

`m 88.59375,193.85498 1.083984,0 q 0.820313,-0.007 ...`

Note how at the beginning there's an 'm' and two coordinate sets. Before this fix a lower case L was correctly added between the two coordinate sets, but after this fix, the lower-L is not added the path produces very odd vectors. 

I though the SVG was non-standard but it rendered perfectly in all places I've tested.
